css/default.css

```diff
@@ -24,6 +24,8 @@ td.category .small, th.category .small	{ font-weight: normal;  }
 }
 tr.spacer			{ background-color: #ffffff !important; color: #000000; height: 5px; }
 
+/* Issue Summary at bug_report_page.php, bug_update_page.php */
+input[type="text"]#summary { width: 100%; }
```

Comment:
Since we identify the field by id, it seems redundant to have `[type="text"]` selector (even `input` is not necessary).

Reply:
OK it's a bit of overhead for CSS processing, but I guess that's negligible.